f/fcs/fn_fcsCommanderOverride.sqf

```diff
@@ -10,8 +10,17 @@ params ["_vehicle","_caller"];
 if (isNull cursorObject) then {
 	["NO TARGET",1] remoteExec ["f_fnc_fcsLocalWarning",commander _vehicle];
 } else {
-	// Get the position of the centre of the targeted object at the time of the override
-	_overrideTarget = (cursorObject modelToWorldWorld (boundingCenter cursorObject));
+	// Get the position of the targeted object at the time of the override
+	private _positionATL = getPosATL cursorObject;
```

Comment:
Out of interest: did you try the `unitAimPosition` [family](https://community.bistudio.com/wiki/unitAimPosition) of commands? Seem like there are "designed" for this use-case, but may also be inconsistent.

Reply:
unitAimPosition is designed for use on units/real vehicles and relies on a memory point in their model. If the memory point isn't present, it uses the object's bounding centre instead, and presumably that has the same potential issues as manually using boundingCentre. It's important for the override to work well with things like buildings and terrain objects, which are unlikely to have the required memory point.